### ISSUE TITLE
Product List View

### ISSUE THREAD

**Issue by Gouthamve** [reactioncommerce/reaction#465]
Currently workflow doesn't work for large collections of products. 
We need a list with bulk edit functions.

Something like this taken from Prestashop platform:

<img width="1070" alt="screen shot 2015-09-25 at 9 57 39 pm" src="https://cloud.githubusercontent.com/assets/7354143/10106133/7fdcf4e8-63d0-11e5-8dfe-da4f540c2804.png">

Depends on #415

**Comment by aaronjudd** [comment]
Nicer version of https://github.com/reactioncommerce/reaction/issues/300

**Comment by alangunning** [comment]
Any guesstimate on product list view implementation?

**Comment by rymorgan** [comment]
Closing in favor of #300
